mono/metadata/mono-perfcounters.c

```diff
@@ -1483,276 +1567,438 @@ typedef struct {
 	int type;
 } CounterCreationData;
 
+TYPED_HANDLE_DECL (CounterCreationData);
+
+static char*
+append (char* cursor, const char* str)
+{
+	int const length = strlen (str) + 1;
+	return (char*)memcpy (cursor, str, length) + length;
+}
+
+static void
+ves_icall_System_Diagnostics_PerformanceCounterCategory_Create_create_temp (
```

Comment:
We usually reserve the `ves_icall_`-style names for icalls.  Static helper functions can have non-prefixed names.

Reply:
Yeah I thought about that, but couldn't converge on any particular shortening. Just remove "ves_icall_" specifically and leave the rest? As well, the name is somewhat informative -- it isn't a top-level icall function, but it is quite close. The names are long, granted. Btw, Ludovic asked for the renames (at the top-level).

Reply:
Yea the rest of the name can be whatever you like - just reserve the `ves_icall_` prefix to funtion as a strong hint that the function is called from the managed side.